apis/metal3.io/v1alpha1/preprovisioningimage_types.go

```diff
@@ -60,15 +60,15 @@ const (
 	ConditionImageError ImageStatusConditionType = "Error"
 )
 
-// PreprovisioningImageStatus defines the observed state of PreprovisioningImage
+// PreprovisioningImageStatus defines the observed state of PreprovisioningImage.
 type PreprovisioningImageStatus struct {
 	// imageUrl is the URL from which the built image can be downloaded.
-	ImageUrl string `json:"imageUrl,omitempty"`
+	ImageUrl string `json:"imageUrl,omitempty"` //nolint:revive,stylecheck
```

Comment:
ideally this field name should have been `ImageURL` as asked by the linters, I was not sure if I should do it here since this will cause an API change. I am up for suggestions.

Reply:
I would suggest at least a separate PR for this if the consensus is to change .

Reply:
For the time being the exclusion is justified IMO.